OrbitGl/CaptureSerializer.cpp

```diff
@@ -83,16 +83,6 @@ void CaptureSerializer::SaveImpl(T& archive) {
     archive(functions);
   }
 
-  {
-    ORBIT_SIZE_SCOPE("Capture::GCallstacks");
-    archive(Capture::GCallstacks);
-  }
-
-  {
-    ORBIT_SIZE_SCOPE("Capture::GProcessId");
```

Comment:
This is used in `SchedulerTrack::UpdatePrimitives` to highlight all and only the scheduling slices that belong to the same process, so it should not be removed until it's replaced by something else.

Reply:
Yes, you're right, thanks for the catch!